client/my-sites/media/main.jsx

```diff
@@ -192,12 +192,14 @@ class Media extends Component {
 	 * @param  {Function} [callback] - callback function
 	 */
 	deleteMedia( callback ) {
-		const site = this.props.selectedSite;
-		const selected = MediaLibrarySelectedStore.getAll( site.ID );
+		const { selectedSite, translate } = this.props;
+		const selected = MediaLibrarySelectedStore.getAll( selectedSite.ID );
 		const selectedCount = selected.length;
-		const confirmMessage = this.props.translate(
-			'Are you sure you want to permanently delete this item?',
-			'Are you sure you want to permanently delete these items?',
+		const confirmMessage = translate(
+			'Are you sure you want to delete this item? \
```

Comment:
It could be that this is fine, but is there any risk to adding newlines to translation strings like this? I can't say I've ever seen it done before, but that may just be my lack of experience with the translation system. Was there a precedent that this is based on?

Looking at [the docs for `translate()`](https://github.com/Automattic/i18n-calypso#strings-only) it seems to suggest concatenating multiple strings using `+` to achieve newlines.

```
// do concatenate long strings with the + operator
var translation = i18n.translate(
    'I am the very model of a modern Major-General, ' +
    'I\'ve information vegetable, animal, and mineral, ' +
    'I know the kings of England, and I quote the fights historical ' +
    'from Marathon to Waterloo, in order categorical.'
);
```